api/models/User.js

```diff
@@ -240,4 +252,96 @@ UserSchema.statics.getUserPictureAboutCategories = function (userId) {
   return this.findById(userId, 'picture about categories').exec();
 };
 
+// Get top ranked user ids for a given investor category, descending
+// Ranking done by total reps
+// Only returns the username, picture, rank, and reps for the category
+UserSchema.statics.findTopRankedInvestors = function(category) {
+  return this.aggregate([
+    { $match: { "portfolio.category": category }},
+    { $unwind: "$portfolio" },
+    { $match: { "portfolio.category": category }},
+    { $sort: { "portfolio.rank": 1 }},
+    { $project: { _id: 1, picture: 1, "portfolio.rank": 1, username: 1 }},
+    { $limit: 10 },
+  ]).exec();
+};
+
+// Get top ranked user ids for a given expert category, descending
+// Ranking done by total reps
+// Only returns the username, picture, rank, and reps for the category
+UserSchema.statics.findTopRankedExperts = function(category) {
+  return this.aggregate([
+    { $match: { "categories.name": category }},
+    { $unwind: "$categories" },
+    { $match: { "categories.name": category }},
+    { $sort: { "categories.rank": 1 }},
+    { $project: { _id: 1, picture: 1, "categories.rank": 1, "categories.reps": 1, username: 1 }},
+    { $limit: 10 },
+  ]).exec();
+};
+
+// Get ranked user ids for a given expert category, descending
+// Ranking done by total reps
+// Only returns the IDs of the users in ranked order
+UserSchema.statics.findRankedExperts = function(category) {
```

Comment:
These queries only return the ids, which is cryptic. This is because these functions are only called when we are recalculating rank. They don't pipe data to the frontend